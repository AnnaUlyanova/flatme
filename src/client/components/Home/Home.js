import React from 'react'

import Landing from './Landing'
import Dashboard from './Dashboard/'

const Home = () => {
  const login = true
  return (
    <div>
      {login ? <Dashboard /> : <Landing />}
    </div>
  )
}

export default Home
