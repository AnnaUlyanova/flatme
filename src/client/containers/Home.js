import { connect } from 'react-redux'

import Home from '../components/Home'

import { createNewFlat } from '../actions'

const mapStateToProps = state => {
  return {
    account: state.account
  }
}

const mapDispatchToProps = (dispatch, ownProps) => {
  return {
    createNewFlat: (flatName) => { return dispatch(createNewFlat(flatName)) }
  }
}

export default connect(
  mapStateToProps,
  mapDispatchToProps
)(Home)
