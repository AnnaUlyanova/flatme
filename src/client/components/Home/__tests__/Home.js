/* eslint-env jest */

import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'
import React from 'react'

import Home from '..'

const account = {}

test('Home renders correctly', () => {
  const wrapper = shallow(<Home account={account} />)
  expect(toJSON(wrapper)).toMatchSnapshot()
})
