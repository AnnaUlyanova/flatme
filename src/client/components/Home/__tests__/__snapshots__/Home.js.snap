exports[`test Home renders correctly 1`] = `
<div>
  <Landing />
</div>
`;
